src/mobile/package.json

```diff
@@ -98,7 +98,8 @@
     "tweetnacl-util": "^0.15.0",
     "util": "^0.10.3",
     "victory-native": "^30.3.0",
-    "vm-browserify": "0.0.4"
+    "vm-browserify": "0.0.4",
+    "@tradle/react-native-http": "^2.0.0"
```

Comment:
We can probably remove `react-native-http` from line 61 if it's basically the same as this package